test/parallel/test-module-isBuiltIn.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { isBuiltIn } = require('module');
+
+// Includes modules in lib/ (even deprecated ones)
+assert(isBuiltIn('http'));
+assert(isBuiltIn('sys'));
+assert(isBuiltIn('node:fs'));
+assert(isBuiltIn('node:test'));
+
```

Comment:
Could we avoid the camel case in the filename? It is not friendly to case insensitive file systems, such as macOS.

Reply:
Hi @hemanth. could you rename test file to kebab casing please? The writing test guide suggest using kebab casing, see https://github.com/nodejs/node/blob/main/doc/contributing/writing-tests.md#naming-test-files.